### DISCUSSION TITLE
[Feature Request] Can't get Models/Characters through Lua for other themes

### DISCUSSION THREAD

**Discussion by EngineMachiner** [TeamRizu/OutFox#264] [03 - Feature Requests]
I can't get the Models in ScreenGameplay. I did see it's possible to get them with Soundwaves.
Any advices to get them?

**Comment by JoseVarelaP** [comment]
The characters in Soundwaves are possible to be obtained as they're being generated on a lua file in ScreenGameplay's background layer. Legacy characters are not possible to be obtained as they are in a separate isolated layer in gameplay and it's not even named, not attached to anything to lua.

What could be done?
We could attach functions to the engine version of DancingCharacters so it can be obtained, but at the same time, you can have way more control by having the characters generated in a lua file. Check soundwaves's `ScreenGameplay background.lua` file for details about that.
